Automattic-Tracks-iOS/LogFile.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public class LogFile {
+    var uuid: String
+    var url: URL
+
+    public init(url: URL, uuid: String = UUID().uuidString) {
+        self.url = url
+        self.uuid = uuid
+    }
```

Comment:
I believe `struct` would be more appropriate for this type. It's lightweight and it's just data. I believe we can also make the properties immutable. 

```suggestion
public struct LogFile {
    let url: URL
    let uuid: String

    init(url: URL, uuid: String = UUID().uuidString) {
        self.url = url
        self.uuid = uuid
    }
}
```

We have a `MockLogFile` subclass in the test which prevents us from changing it to a `struct` but I'd suggest that we don't need that `MockLogFile` at all. More about that later. 🙂

Reply:
Yep – agreed, and this was changed in fd80503b343b7a124e364ca4bde5fd4c14148b61.